kafka-init/src/main/java/io/strimzi/kafka/init/InitWriterConfig.java

```diff
@@ -40,6 +47,14 @@ public class InitWriterConfig {
      * The address type which should be preferred in the selection
      */
     public static final ConfigParameter<String> EXTERNAL_ADDRESS_TYPE = new ConfigParameter<>("EXTERNAL_ADDRESS_TYPE", STRING, null, CONFIG_VALUES);
+    /**
+     * FWSS secret prefix to filter secrets
+     */
+    public static final ConfigParameter<String> FWSS_SECRETS_PREFIX = new ConfigParameter<>("FWSS_SECRETS_PREFIX", STRING, "fwss", CONFIG_VALUES);
```

Comment:
can we get this as a configurable value from a environment variable , passed through a config map